app/views/home/_card.html.erb

```diff
@@ -1,5 +1,5 @@
 <article class="card">
-  <a class="card__link" href="#">
+  <a class="card__link" href="<%= post.path %>">
```

Comment:
The PostController is under the Dashboard namespace, only logged in users can operate posts.
The method `post_path(post)` does not exist. The only method available is `dashboard_post_path(post)`,  which generate 'dashboard/posts/:slug'.

That's why I think we should made a layer of abstraction before the model. I'm going to move to a helper method.
```ruby
def post_path(post)
  "posts/#{post.friendly_id}"
end
```
But I still think this is not the best solution.